pymeasure/display/Qt.py

```diff
@@ -24,13 +24,18 @@
 
 import logging
 
-from pyqtgraph.Qt import QtGui, QtCore, loadUiType
+from pyqtgraph.Qt import QtGui, QtCore, QtWidgets, loadUiType
 
 log = logging.getLogger(__name__)
 log.addHandler(logging.NullHandler())
 
 QtCore.QSignal = QtCore.Signal
```

Comment:
I came across this line, does anyone know what it's use is? Why not replace all usage in our codebase of `QtCore.QSignal` by `QtCore.Signal` and drop this line?

Reply:
Git blame says that this was changed by @msmttchr in #431 to add pyside2 support. I don't know why, though.

Reply:
I think what suggested by @CasperSchippers is doable.
I think I made this change to keep minimal modifications or simply I didn't realize that this line could have been removed. If I get it right, pyqtgraph provides `QtCore.Signal` for both `PyQt5` and `PySide2`

Reply:
Thanks, I removed the line